tests/common/platform/conn_graph_facts.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import os
+
+@pytest.fixture(scope="module")
+def conn_graph_facts(testbed_devices):
```

Comment:
Just thinking out loud, but do you think this should be part of platform or something more general? I'm admittedly not super familiar with this part of testbed so I'm not sure *how* general this is, it just seems like something we'd want to utilize in a wide range of test suites.

Reply:
I agree that this could be more general than just platform.